handler/interface.go

```diff
@@ -16,6 +16,7 @@ type responder interface {
 
 type cantabularClient interface {
 	GetCodebook(context.Context, cantabular.GetCodebookRequest) (*cantabular.GetCodebookResponse, error)
+	GetGeographyDimensions(ctx context.Context, dataset string) (*cantabular.GetGeographyDimensionsResponse, error)
```

Comment:
why are the parameters named here but not in the line above?

Reply:
`GetCodebook` is very clear with the parameter types, but `GetGeographyDimensions` accepts a string paramter, and if we don't provide the name we wouldn't know what it is.
I can name all parameters for consistency.